CMakeLists.txt

```diff
@@ -388,20 +392,20 @@ else()
 		set(DATA_DIR "." CACHE STRING "Where to install data files")
 	else()
 		if(NOT BIN_DIR)
-			set(BIN_DIR "bin" CACHE STRING "Where to install binaries")
+			set(BIN_DIR "${CMAKE_INSTALL_PREFIX}/bin" CACHE STRING "Where to install binaries")
```

Comment:
CMake script later uses BIN_DIR for `install()` command, meaning that BIN_DIR (and others) should stay without CMAKE_INSTALL_PREFIX.
I also don't see how this could have effect on ENABLE_MONOLITHIC_INSTALL since these definitions are explicitly in else() block and can only be reached if ENABLE_MONOLITHIC_INSTALL is NOT set.

(same issue with LIB_DIR & DATA_DIR)

I think what you want to do is to call `add_definitions` separately for monolithic install and for standard install